### DISCUSSION TITLE
Convert Hex to ACI color

### DISCUSSION THREAD

**Discussion by Lukideluki** [mozman/ezdxf#609] [Q&A]
Hi, i am looking a way to convert hex colors back to the ACI color.

I used the follow code to convert ACI into Hex:
`        aci = entity.dxf.color
        rgb24 = colors.DXF_DEFAULT_COLORS[aci]
        color= f"#{rgb24:06X}"   `
now i am tring to do the reverse but didnt found the way.

**Comment by mozman** [comment]
You can do a reverse search, but if your RGB value is not exact a default ACI color, it's getting complicated. Some key words: perceived color, luminance, color spaces ...

A possible solution is shown here:
https://stackoverflow.com/questions/1847092/given-an-rgb-value-what-would-be-the-best-way-to-find-the-closest-match-in-the-d
